pkg/controller/authenticationservice/standard.go

```diff
@@ -111,6 +112,9 @@ func applyStandardAuthServiceDeployment(authservice *adminv1beta1.Authentication
 		jvmOptions := "-Dvertx.cacheDirBase=/tmp -Djboss.bind.address=0.0.0.0 -Djava.net.preferIPv4Stack=true -Duser.timezone=UTC"
 		if authservice.Spec.Standard.JvmOptions != nil {
 			jvmOptions += " " + *authservice.Spec.Standard.JvmOptions
+		} else if qty, ok := authservice.Spec.Standard.Resources.Requests["memory"]; ok {
+			maxHeap := qty.ScaledValue(resource.Mega)
```

Comment:
Poor variable name.  It is not (JVM) Heap, it is the container's memory.

Reply:
Fixed